docs/reference/pages/model_recipes.md

```diff
@@ -276,4 +276,4 @@ class TagsSnippetViewSet(SnippetViewSet):
 register_snippet(TagsSnippetViewSet)
 ```
 
-A `Tag` model has a `name` and `slug` required fields. If you decide to add a tag, it is recommended to only display the `name` field panel as the slug field is autofilled when the `name` field is filled and you don't need to enter the same name in both the fields.
+A `Tag` model has a `name` and `slug` required fields. If you decide to add a tag, it is recommended to only display the `name` field panel as the slug field is auto-filled when the `name` field is filled and you don't need to enter the same name in both fields.
```

Comment:
```suggestion
A `Tag` model has a `name` and `slug` required fields. If you decide to add a tag, it is recommended to only display the `name` field panel as the slug field is automatically populated when the `name` field is filled and you don't need to enter the same name in both fields.
```

Not sure on the word but we maybe should avoid the term autofill/auto-fill as it could be confused with browser auto-fill/autocomplete.